lib/docs/stories/introduction/usage.stories.mdx

```diff
@@ -1,42 +1,63 @@
+- [**Message list**](https://pubnub.github.io/react-chat-components/docs/?path=/docs/components-member-list--default) that will show when you start typing and sending messages
 
 ## Requirements
 
-- React v16.8+ and ReactDOM v16.8+
+You will use the following tools in the course of this guide:
+
+- [React v16.8+](https://www.npmjs.com/package/react/v/16.8.0) and [ReactDOM v16.8+](https://www.npmjs.com/package/react-dom)
 - [PubNub JavaScript SDK v4.29+](https://www.pubnub.com/docs/sdks/javascript/)
 - [PubNub React SDK v2.1.0+](https://www.pubnub.com/docs/chat/react/setup)
+- Code editor of your choice (e.g. [Visual Studio Code](https://code.visualstudio.com/download))
+
+## Steps
+
+Let's get started!
+
+### Create a PubNub account
+
+Before you start, you need to get **Publish Key** and **Publish Key** for your chat app.
+Sign in or create an account on the [Admin Portal](https://dashboard.pubnub.com/) to create an app and a new keyset. 
 
-## Set up PubNub account
+>**TIP**: When you create a new app on the Admin Portal, the first set of keys is generated automatically, but a single app can have as many keysets as you like. We recommend that you create separate keysets for production and test environments. There are some of the functionalities you might need to enable on your keyset depending on the use case, but they won't be required to complete this Getting Started guide. These options include:
+   - [**PRESENCE**](https://www.pubnub.com/docs/sdks/javascript/api-reference/presence) to monitor the subscribers of channels and delivers information on their real-time status
+   - [**STORAGE & PLAYBACK**](https://www.pubnub.com/docs/sdks/javascript/api-reference/storage-and-playback) (including correct **Retention**) to persist all messages as they're published.
+   - [**OBJECTS**](https://www.pubnub.com/docs/sdks/javascript/api-reference/objects) to easily store metadata about your application users and channels, and their membership associations, without the need to stand up your own databases. When using this feature, make sure you select a geographical region corresponding to most users of your application and to have these enabled: **User Metadata Events**, **Channel Metadata Events**, and **Membership Events**.
 
-1. Sign in or create an account to create an app on the
-   [Admin Portal](https://dashboard.pubnub.com/) and get the keys to use in your application.
+### Create a new project for your app
 
-2. When you create a new app, the first set of keys is generated automatically, but a single app can
-   have as many keysets as you like. We recommend that you create separate keysets for production
-   and test environments.
+Create a sample React project you will use as a placeholder for your chat app code. 
+To use the [`react-sample-chat`](https://www.pubnub.com/docs/sdks/javascript/react/#download-the-react-wrapper) app, follow these steps:
```

Comment:
Does it make sense to start with an app that already has a working Chat? Perhaps we should start with an empty app? I can provide instructions for that

Reply:
Improved that point